priv/worker.py

```diff
@@ -33,7 +49,7 @@
     year = {2016},
     publisher = {GitHub},
     journal = {GitHub repository},
-    howpublished = {\url{https://github.com/openai/gym-http-api}}
+    howpublished = {\\url{https://github.com/openai/gym-http-api}}
```

Comment:
okay, let's go with ``howpublished = {https://github.com/openai/gym-http-api}`` for now.